hypothesis-python/src/hypothesis/core.py

```diff
@@ -1362,6 +1375,11 @@ def _raise_to_user(errors_to_report, settings, target_lines, trailer=""):
     if settings.verbosity >= Verbosity.normal:
         for line in target_lines:
             add_note(the_error_hypothesis_found, line)
+
+    if verified_by:
+        msg = f"backend={verified_by!r} verified this test passes - please report that as a bug!"
```

Comment:
```suggestion
        msg = f"backend={verified_by!r} claimed to verify this test passes - please report that as a bug!"
```

maybe? to make it clear that the verification is at fault